### PR TITLE
Add company attachment endpoints to B2B Edition API specs

### PR DESCRIPTION
**Pull request by becomevocal**
## What changed?
- Moved over B2B endpoints for Company Attachments that launched since the initial docs transfer into our dev center
- Improved some of the descriptions in the spec
- Removed unnecessary x-stoplight references

ping @bigcommerce/dev-docs-team @bigcommerce/team-b2b

### CODE REVIEW

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 1]
```diff
@@ -129,7 +143,7 @@ paths:
             type: integer
           in: query
           name: customerId
-          description: 'BigCommerce customer id, this customer should be a B2B company user'
+          description: "BigCommerce customer id, this customer should be a B2B company user"
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:146]:
```suggestion
          description: "BigCommerce customer ID. This customer should be a B2B company user."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 2]
```diff
@@ -138,9 +152,9 @@ paths:
             default: search
           in: query
           name: extraFieldFilterType
-          description: 'extra field filter type. The search filter type allows for partial matches, while the exact-match filter type only returns results that exactly match the specified criteria.'
+          description: "extra field filter type. The search filter type allows for partial matches, while the exact-match filter type only returns results that exactly match the specified criteria."
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:155]:
```suggestion
          description: "Extra field filter type. The search filter type allows for partial matches, while the exact-match filter type only returns results that exactly match the specified criteria."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 3]
```diff
@@ -516,7 +530,7 @@ paths:
                         type: integer
                     originChannelId:
                       type: integer
-                      description: 'BigCommerce channel id, used for BigCommerce customer origin channel id. This field takes effect only when the store default b2b channel is not configured'
+                      description: "BigCommerce channel id, used for BigCommerce customer origin channel id. This field takes effect only when the store default b2b channel is not configured"
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:533]:
```suggestion
                      description: "BigCommerce channel ID, used for BigCommerce customer origin channel ID. This field takes effect only when you do not configure the store default b2b channel."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 4]
```diff
@@ -620,12 +634,12 @@ paths:
                     data: {}
                     meta:
                       message: Create BC customer group failed
-        '422':
-          description: 'Company was not valid. This is the result of missing required fields, or of invalid data. See the response for more details'
+        "422":
+          description: "Company was not valid. This is the result of missing required fields, or of invalid data. See the response for more details"
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:638]:
```suggestion
          description: "Company was not valid. This is the result of missing required fields, or of invalid data. See the response for more details."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 5]
```diff
@@ -1430,17 +1444,17 @@ paths:
...
                   errors:
-                    - id: '42'
-                      detail: ''
+                    - id: "42"
+                      detail: ""
                 meta:
-                  message: 'Some companies updated failed, you can see these in errors'
-        '409':
-          description: 'Company was in conflict with another company. This is the result of duplicate unique values; A missing or invalid `companyEmail`, `phoneNumber` or other extraFields'
+                  message: "Some companies updated failed, you can see these in errors"
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:1450]:
```suggestion
                  message: "Some companies updated failed, you can see these in errors."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 6]
```diff
@@ -1595,7 +1609,7 @@ paths:
                           type: integer
                       originChannelId:
                         type: integer
-                        description: 'BigCommerce channel id, used for BigCommerce customer origin channel id. This field takes effect only when the store default b2b channel is not configured'
+                        description: "BigCommerce channel id, used for BigCommerce customer origin channel id. This field takes effect only when the store default b2b channel is not configured"
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:1612]:
```suggestion
                        description: "BigCommerce channel ID, used for BigCommerce customer origin channel ID. This field takes effect only when you do not configure the store default b2b channel."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 7]
```diff
@@ -2326,12 +2290,12 @@ paths:
                   description: Get addresses
                   code: get_addresses
                   moduleName: address
-        description: 'name, code are unique in the same store.'
+        description: "name, code are unique in the same store."
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:2293]:
```suggestion
        description: "name, code is unique in the same store."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 8]
```diff
@@ -2457,15 +2393,15 @@ paths:
                   description: Get addresses
                   code: get_addresses
                   moduleName: address
-        description: 'name, code are unique in the same store'
+        description: "name, code are unique in the same store"
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:2396]:
```suggestion
        description: "name, code is unique in the same store."
```

**Review by bc-traciporter — COMMENTED** [review]
Removed passive voice and other editorial comments

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 9]
```diff
@@ -1,24 +1,30 @@
...
   - name: Company Roles and Permissions
 paths:
   /companies:
     get:
       tags:
         - Company
-      summary: Get Companies
-      description: Get all companies list
+      summary: Get All Companies
+      description: Get a list of all companies
```
Comment (bc-andreadao) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:27]:
```suggestion
      description: Get a list of all companies.
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 10]
```diff
@@ -60,7 +66,7 @@ paths:
             type: number
         - name: sortBy
           in: query
-          description: The response sorted by which field
+          description: Sort the result by a specific company field
```
Comment (bc-andreadao) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:69]:
```suggestion
          description: Sort the result by a specific company field.
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 11]
```diff
@@ -80,16 +86,16 @@ paths:
             example: DESC
         - name: isIncludeExtraFields
           in: query
-          description: Is show extra fields in the response
+          description: Should extra fields be shown in the response
```
Comment (bc-andreadao) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:89]:
```suggestion
          description: Whether the response will include extra fields.
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 12]
```diff
@@ -507,7 +521,7 @@ paths:
                       description: BC price list Id that company related
                     acceptCreationEmail:
                       type: boolean
-                      description: 'True: Will send email company creation notification to admin user; False: Will not send'
+                      description: "True: Will send email company creation notification to admin user; False: Will not send"
```
Comment (bc-andreadao) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:524]:
```suggestion
                      description: "True: Sends an email notification about the company creation to the admin user; False: Does not send the notification."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 13]
```diff
@@ -654,12 +668,12 @@ paths:
...
       x-codegen-request-body-name: body
-  '/companies/{companyId}':
+  "/companies/{companyId}":
     get:
       tags:
         - Company
-      summary: Get a Company Detail
-      description: Get a company detail
+      summary: Get a Company
+      description: Get details about a specific company
```
Comment (bc-andreadao) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:676]:
```suggestion
      description: Get details about a specific company.
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml [hunk 14]
```diff
@@ -1430,17 +1444,17 @@ paths:
...
-                      detail: ''
+                    - id: "42"
+                      detail: ""
                 meta:
-                  message: 'Some companies updated failed, you can see these in errors'
-        '409':
-          description: 'Company was in conflict with another company. This is the result of duplicate unique values; A missing or invalid `companyEmail`, `phoneNumber` or other extraFields'
+                  message: "Some companies updated failed, you can see these in errors"
+        "409":
+          description: "Company was in conflict with another company. This is the result of duplicate unique values; A missing or invalid `companyEmail`, `phoneNumber` or other extraFields"
```
Comment (bc-andreadao) [inline docs/b2b-edition/specs/api-v3/company/company.yaml:1452]:
```suggestion
          description: "Company was in conflict with another company. This is the result of duplicate unique values; A missing or invalid `companyEmail`, `phoneNumber` or other extraFields."
```

File: docs/b2b-edition/specs/api-v3/company/company.yaml (hunk 7 above)
Comment (bc-andreadao) [reply docs/b2b-edition/specs/api-v3/company/company.yaml:2293]: I actually think it should be:

The name and code are unique within the same store.

File: docs/b2b-edition/specs/api-v3/company/company.yaml (hunk 8 above)
Comment (bc-andreadao) [reply docs/b2b-edition/specs/api-v3/company/company.yaml:2396]: The name and code are unique within the same store.

**Review by bc-andreadao — APPROVED** [review]

**Review by bc-traciporter — APPROVED** [review]

**Pull request merged**